LEAF_Request_Portal/templates/view_reports.tpl

```diff
@@ -852,26 +925,39 @@ $(function() {
                 }
             	grid.loadData(recordIDs);
             }
-    	});
+            let results = grid.getCurrentData().slice();
```

Comment:
Does recreating a shallow copy for filtering really save a lot of time?

Reply:
I took this out, since the filter should take care of this anyway.